cdap-ui/app/cdap/components/AbstractWidget/InputFieldDropdown/index.tsx

```diff
@@ -18,16 +18,24 @@ import Select from 'components/AbstractWidget/FormInputs/Select';
 import { IWidgetProps, IStageSchema } from 'components/AbstractWidget';
 import { objectQuery } from 'services/helpers';
 import { WIDGET_PROPTYPES } from 'components/AbstractWidget/constants';
+import MultiSelect from '../FormInputs/MultiSelect';
 
 interface IField {
   name: string;
-  type: string;
+  type: string[];
 }
 
-interface IInputFieldProps extends IWidgetProps<null> {}
+const delimiter: string = ',';
```

Comment:
nit: ts can infer primitive type if we assign it with a value in the same line. The string type definition is just redundant here

Reply:
Done